script/deploy/deployUpgradesToSAFE.sh

```diff
@@ -31,24 +31,9 @@ deployUpgradesToSAFE() {
     CLEAN_RETURN_DATA=$(echo $RAW_RETURN_DATA | sed 's/^.*{\"logs/{\"logs/')
     FACET_CUT=$(echo $CLEAN_RETURN_DATA | jq -r '.returns.cutData.value')
     if [ "$FACET_CUT" != "0x" ]; then
-      CUTS+=("$FACET_CUT")
+      echo "Proposing facet cut for $script..."
+      ts-node script/deploy/safe/propose-to-safe.ts --to "$DIAMOND_ADDRESS" --calldata "$FACET_CUT" --network "$NETWORK" --rpcUrl $(getRPCUrl $NETWORK) --privateKey "$PRIVATE_KEY"
```

Comment:
Would it make sense to propose upgrades as a dev instead of using the deployer `PRIVATE_KEY` here. I think another env variable for the developers key would make sense.